packages/langium/src/default-module.ts

```diff
@@ -33,6 +34,9 @@ export type DefaultModuleContext = {
 export function createDefaultModule(context: DefaultModuleContext = {}): Module<LangiumServices, LangiumDefaultServices> {
     return {
         parser: {
+            GrammarConfig: () => ({
+                multilineCommentRules: ['ML_COMMENT']
+            }),
```

Comment:
We should consider specifying comment rules in a way that the CLI can use that information as well, e.g. for #204. That could be through a grammar language extension (see #185) or through JSON configuration. WDYT?

Reply:
I don't like either approach, but I propose a third. Identify whether the token is a comment by two properties:

1. It's a hidden token
2. It doesn't match a whitespace string

A similar approach is already used in the parser interpreter which sorts terminals depending on whether they're able to match a whitespace. Of course, we can always let the user override the computed grammar config. Do you think that's a path we can take?

Reply:
Yes, I think it's totally fine to assume that all tokens that are hidden but contain non-whitespace characters are comments.

Reply:
Alright, I would tackle this in a different issue. This PR should be ready for a last review 👍